app/components/primer/beta/border_box.pcss

```diff
@@ -59,7 +62,7 @@
   & .Box-btn-octicon {
     &.btn-octicon {
       padding: var(--stack-padding-spacious);
-      margin: calc(var(--stack-gap-spacious) * -1) calc(var(--stack-gap-spacious) * -1);
+      margin: calc(var(--stack-gap-spacious) * -1);
```

Comment:
this change looks sus

Reply:
I ran into a rule about shorthand duplication, since the 2 values for top and sides was the same